src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -29,14 +27,15 @@ interface RoomModalProps {
 }
 
 export const RoomModal: React.FC<RoomModalProps> = ({ show, onHide, room }) => {
+  useFirestoreConnect("venues");
```

Comment:
This will load the data for all venues in the database, is that really the desired outcome here?

What prompted this change out of curiosity? It seems unrelated to the other changes being made in this PR.

Reply:
venueEvents are empty array when venues are not loaded.

Reply:
I don't think these two are related since `venueEvents` is a separate entity with its own useFirestoreConnect.